docs/tutorials/tutorials_intro.md

```diff
@@ -1,3 +1,19 @@
 ## Arm Mbed tutorials
 
-The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides step-by-step instructions for specific tasks you may wish to perform. These include Blinky, our example application; applications that build on Blinky; and debugging tasks. There are also tutorials about using the Arm Mbed Online Compiler, optimizing memory and optimizing power.
+The Arm Mbed ecosystem is expansive and offers many opportunities. In contrast to other sections of the documentation, which provide background reference material, this section provides step-by-step instructions for specific tasks you may wish to perform. These include our quick start guide, example applications that build on it and debugging tasks. There are also tutorials about using the Arm Mbed Online Compiler, optimizing memory and optimizing power.
+
+The [Mbed OS quick start](mbed-os-quick-start.html) is our quick start guide, a tutorial that shows you how to set up and run an example application.
+
+[Major releases](major-releases.html) discusses the difference between Mbed OS 2 and Mbed OS 5 and answers frequently asked questions about using the two major versions of Mbed OS. This section also teaches you how to migrate projects to Mbed OS 5.
+
+[Using the Mbed Online Compiler](using-the-mbed-online-compiler.html) uses screenshots and detailed instructions to teach you how to use our online IDE. It guides you through the process of creating, developing and publishing a process and includes advanced steps, such as collaborating with multiple authors.
+
+[Serial communication](serial-communication.html) teaches you how to install a serial driver, communicate with your board over USB and program and debug with terminal applications.
+
+The [Using APIs](using-apis.html) shows you how to use our APIs to perform specific tasks. From creating an event loop to building a private LoRa network, these tutorials include RTOS APIs, drivers and connectivity APIs.
```

Comment:
Either remove "the" or add a word after "using APIs"

Reply:
My concern is that a lot of the wording in this intro is very specific to the current state of the docs and you'll need to remember to rewrite every time you add or remove something